resources/profiles/Qidi/filament/Qidi Generic ABS @Qidi XMax3 0.2 nozzle.json

```diff
@@ -0,0 +1,32 @@
+{
+    "type": "filament",
+    "filament_id": "GFB99",
+    "setting_id": "GFSA04",
+    "name": "Qidi Generic ABS @Qidi XMax3 0.2 nozzle",
```

Comment:
the machine name after @ should match the actual machine name.
So here both the file name and name should be `Qidi Generic ABS @Qidi X-Max 3 0.2 nozzle`
Otherwise Orca won't recognize this as a specialized profiles of Qidi Generic ABS